frontend/elements/src/core/titles/variants/title-w-icon.ts

```diff
@@ -37,24 +37,24 @@ export class _ extends LitElement {
   path:string = ""; 
   @property({type: Boolean})
   uploaded: boolean = false;
+  
   @property({type: Boolean})
   hover:boolean = false;
   
   onHover() {
```

Comment:
This isn't working - please see the solution shared by Daniel at https://polymer.slack.com/archives/C03PF4L4L/p1611233959006300

Reply:
This will be fixed by David